frontend/src/scenes/funnels/funnelLogic.ts

```diff
@@ -273,6 +286,18 @@ export const funnelLogic = kea<funnelLogicType>({
                 setIsGroupingOutliers: (_, { isGroupingOutliers }) => isGroupingOutliers,
             },
         ],
+        visibilityMap: [
```

Comment:
A comment clarifying what this reducer is for could help understand better what it's used for